ledger/complete/mtrie/trie/trie_update_test.go

```diff
@@ -0,0 +1,68 @@
+package trie_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/common/testutils"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
+	"github.com/stretchr/testify/require"
+)
+
+// adding payloads to the trie until full
+func Test_AddingPayloadUntilFull(t *testing.T) {
+
+	// prepare paths for a trie with only 2 levels, which has 4 payloads in total
+	paths := []ledger.Path{
+		testutils.PathByUint8(0),           // 00000000...
+		testutils.PathByUint8(1 << 6),      // 01000000...
+		testutils.PathByUint8(1 << 7),      // 10000000...
+		testutils.PathByUint8(1<<7 + 1<<6), // 11000000...
+	}
+
+	// prepare payloads, the value doesn't matter
+	payloads := make([]ledger.Payload, 0, len(paths))
+	for i := range paths {
+		payloads = append(payloads, *payloadFromInt16(uint16(i)))
+	}
+
+	t0 := trie.NewEmptyMTrie()
+	fmt.Println(t0)
```

Comment:
I think we don't need all the prints, but if so, probably should use test logger so it can be managed